minio/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.7"
+
+services:
+    app_proxy:
+        environment:
+            APP_HOST: minio_web_1
+            APP_PORT: 9011
+            PROXY_AUTH_ADD: "false"
+
+    web:
+        image: "quay.io/minio/minio:RELEASE.2024-02-17T01-15-57Z"
+        restart: on-failure
```

Comment:
Whenever possible we try to run applications as a non-root user. It looks like minIO supports the default docker behaviour, so we can add:

```
user: "1000:1000"
```